src/main/java/uk/ac/ebi/eva/pipeline/jobs/deciders/EmptyVepOutputDecider.java

```diff
@@ -64,10 +65,14 @@ private String getVepInput(JobExecution jobExecution) {
     private long getFileSize(String file) {
         long fileSize;
 
-        try {
-            fileSize = Files.size(Paths.get(file));
-        } catch (IOException e) {
-            throw new RuntimeException("File {} is not readable", e);
+        if (!new File(file).exists()) {
+            fileSize = 0;
+        } else {
+            try {
+                fileSize = Files.size(Paths.get(file));
+            } catch (IOException e) {
+                throw new RuntimeException("File " + file + " is not readable", e);
```

Comment:
Would it be that inconvenient to wrap the IOException with another (checked) IOException? If it is not readable, we want to make sure the issue is properly reported, instead of the exception re-thrown indefinitely.